scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl

```diff
@@ -362,11 +404,6 @@ task PrepVtAnnotationJson {
         String output_file_suffix
         String output_path
     }
-    parameter_meta {
```

Comment:
Unclear to me how this could have worked without localization of the annotation json.

Reply:
same--I know Miguel was working on something but I agree that we should skip for now